va-gov/pages/housing-assistance/index.md

```diff
@@ -104,27 +104,27 @@ Whatever you’re struggling with—homelessness, chronic pain, anxiety, depress
   <h2>Get VA Home Loan Benefits</h2>
```

Comment:
Can you get the updated cross links in the top section?

Reply:
@RobAWilkinson ?

Reply:
Sorry, the cross links need to be updated to match the spreadsheet here as well.
The spreadsheet has:
Disability Compensation (Pay)
Pension Benefits
Vocational Rehabilitation and Employment
Life Insurance

The crosslinks section here is missing the last 2